index.js

```diff
@@ -31,10 +31,11 @@ const cmds = new Map([
 	['setYear', (dt, y) => dt.set({year: y})],
 
 	['startOfWeek', dt => dt.startOf('week')],
+	['startOfMonth', dt => dt.startOf('month')],
 	['endOfWeek', dt => dt.endOf('week')]
 ])
 
-const createParse = (DateTime) => {
+const createParse = DateTime => {
 	const parseHumanRelativeTime = (str, now = DateTime.local()) => {
```

Comment:
Please give the Luxor `parseHumanRelativeTime` variant the same signature as the date-fns variant, by adding `opt = {}`.